### ISSUE TITLE
[Bug]: In multi-column mode, editor panes grow to fit the content of long text values in the editor searc/replace text fields.

### ISSUE THREAD

**Issue by Foxcapades** [mallowigi/material-theme-issues#520]
### Describe the bug

When a long text value is typed, pasted, or selected from history into one of the editor search/replace text fields, the input field will expand to try and fit the width of the input text.  If the input text is wide enough, the text field's expansion will cause the parent editor pane itself to begin expanding in width up to a maximum point.

Compounding on [a UI bug in the IntelliJ Platform itself](https://youtrack.jetbrains.com/issue/IJPL-155881/), the pane's width cannot be reduced from the automatically expanded width.

This may be tedious to debug/fix as it is tangled up in the platform's bug, however the behavior with the Material Theme UI is different than the behavior of the platform's bug.

Changes to the existing 'bad' behavior with the plugin enabled:

* The parent editor pane resizes to fit the input field width (this does not happen without the plugin)
* The input field width cannot be adjusted at all, disabling the workaround available for the platform bug

Due to those changes, I have not yet found a workaround to enable resizing the editor pane to make a multi-column layout workable with long search/replace values.

### To Reproduce

From a clean slate (no open editors)

1. Open a text-based file in an editor.
2. Right click the editor tab/title and select 'Split Right' to enable a multi-column editor layout.
3. In either open editor pane, open the search (or search & replace) panel.
4. Type, select from history, or paste a long text value into one of the search/replace text input fields.

### Expected behavior

The input field should grow in width to try and fit the width of the input text, to a maximum value that does not impact the width of the parent editor pane.

In other words, the automatic width increase of the input fields should stop before causing the width of the parent editor pane to change.

### Material Theme Version

v9.3.1

### IDE

IntelliJ IDEA, PHPStorm, Rider, PyCharm, Other (AppCode, CLion, GoLang...)

### System Info

I have tested on all the JetBrains IDEs i have installed locally (they were all 2024.1) and it is present in all of them.  I then updated all of them to 2024.1.2, and the behavior is still present.

* OS: Ubuntu 23.10

### Logs/Screenshots

[![](https://img.youtube.com/vi/ZpqoWHXlft0/0.jpg)](https://youtu.be/ZpqoWHXlft0)

**Comment by mallowigi** [comment]
will be fixed on the next release but only on the 2024.2 versions
